### PR TITLE
Play Takeoff/Landing sounds for Planes as well as Helicopters

### PR DESCRIPTION
**Pull request by phrohdoh**
Written to fix https://github.com/OpenRA/OpenRA/pull/9891#issuecomment-154857048.

### CODE REVIEW

**Comment by obrakmann** [comment]
Thanks. Any reason why you added it `Fly` instead of `TakeOff`, though?

**Comment by Phrohdoh** [comment]
Oh, only because the code I copied existed in `HeliFly`, I can change it if that is more correct.

**Comment by penev92** [comment]
I agree it would fit better in `TakeOff`.

**Comment by Phrohdoh** [comment]
Moved into `TakeOff`.

**Comment by abcdefg30** [comment]
This doesn't build at the moment.

File: OpenRA.Mods.Common/Activities/Air/TakeOff.cs [hunk 1]
```diff
@@ -30,6 +31,12 @@ public override Activity Tick(Actor self)
 			if (NextActivity == null)
 				self.CancelActivity();
 
+			if (!playedSound && plane.Info.TakeoffSound != null && self.IsAtGroundLevel())
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Activities/Air/TakeOff.cs]: I think it's `aircraft` instead of `plane`.

Comment (Phrohdoh) [inline OpenRA.Mods.Common/Activities/Air/TakeOff.cs]: Correct. I didn't even test this. (shame!)

**Comment by Phrohdoh** [comment]
Fixed.

**Comment by abcdefg30** [comment]
I still somehow don't get a takeoff sound.

**Comment by Phrohdoh** [comment]
Neither do I, now.
I will investigate this more tomorrow.

**Pull request closed without merging**